build.gradle.kts

```diff
@@ -9,10 +9,11 @@ plugins {
     kotlin(Config.cocoapods).version(Config.kotlinVersion).apply(false)
     id(Config.jetpackCompose).version(Config.composeVersion).apply(false)
     id(Config.androidGradle).version(Config.agpVersion).apply(false)
+    id(Config.BuildPlugins.buildConfig).version(Config.BuildPlugins.buildConfigVersion).apply(false)
```

Comment:
Indentation here is off

Reply:
comment is resolved but Indentation is still off

Reply:
hmm it looks fine on my end or do you mean that it needs line breaks due to the length?

Reply:
https://github.com/getsentry/sentry-kotlin-multiplatform/blob/df6bce7dfe31a168e682abeda3e927ed7ac12ca0/build.gradle.kts#L12

Reply:
I guess its the code diff on github, here its clearly off but opening the commit, it is right indeed.